### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by tomii955** [demyte/language-cshtml#3]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:565:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:187:43)
```

**Comment by JaKXz** [comment]
Duplicate of #2 obviously, but +1, using Atom v0.190

**Comment by demyte** [comment]
Hi @tomii955 & @JaKXz 

Thanks for logging this - I am not able to look at this till next week as I am on vacation.

Will sort it out when I am back.

Cheers

James.

**Comment by willl** [comment]
All you need to do is rename the scoped-properties to settings and it should work fine.

**Comment by demyte** [comment]
Pushed up a fix (d6b88ad59dae1aecf48a862d27f47499d1cc0132) and released an updated package
